### PR TITLE
Make Copilot settings real settings

### PR DESCRIPTION
**Pull request by alexr00**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -461,6 +461,21 @@
...
+				},
+				"githubPullRequests.experimental.notificationsView": {
+					"type": "boolean",
+					"markdownDescription": "%githubPullRequests.experimental.notificationsView.description%",
+					"default": false
+				},
+				"githubPullRequests.experimental.useQuickChat": {
+					"type": "boolean",
+					"markdownDescription": "%githubPullRequests.experimental.useQuickChat.description%",
+					"default": true
```
Comment (lszomoru) [inline package.json:477]: Let's flip this one to false since the Chat panel is moving to the auxiliary sidebar

File: package.nls.json [hunk 2]
```diff
@@ -86,6 +86,9 @@
 	"githubPullRequests.createDefaultBaseBranch.repositoryDefault": "The default branch of the repository",
 	"githubPullRequests.createDefaultBaseBranch.createdFromBranch": "The branch that the current branch was created from, if known",
 	"githubPullRequests.createDefaultBaseBranch.auto": "When the current repository is a fork, this will work like \"repositoryDefault\". Otherwise, it will work like \"createdFromBranch\".",
+	"githubPullRequests.experimental.chat.description": "Enables the `@githubpr` Copilot chat provider in the chat view. `@githubpr` can help search for issues and pull requests, suggest fixes for issues, and summarize issues, pull requests, and notifications.",
```
Comment (lszomoru) [inline package.nls.json:89]: Is it chat provider or chat participant?

File: package.json (hunk 1 above)
Comment (aiday-mar) [reply package.json:477]: There are also some rendering issues when running the commands in the quick chat. The response is streamed but it disappears sometimes from the quick chat. Does not seem to happen in panel chat.

File: package.nls.json (hunk 2 above)
Comment (alexr00) [reply package.nls.json]: Thanks for catching!

**Pull request merged**